### DISCUSSION TITLE
PR for Github Education 2022 Grad Ceremony failing even though I have SDP

### DISCUSSION THREAD

**Discussion by thomaslu3** [community/community#17795] [GitHub Education]
I have SDP, I just graduated in December 2021 (but am still in the class of 2022). Can this be resolved?

**Comment by juanpflores** [comment]
Hi @thomaslu3 👋🏼 
thanks for opening this discussion. Went ahead and manually approved your pull request. It should be merged in the next batch. congratulations on graduating ⭐

**Reply by Ramcodeswithbat** [reply]
why dont you help me
